fink_science/kilonova/processor.py

```diff
@@ -157,73 +132,30 @@ def knscore(jd, fid, magpsf, sigmapsf, jdstarthist, cdsxmatch, ndethist, model_p
         'FLT': fid[mask].explode().replace({1: 'g', 2: 'r'})
     })
 
-    # Load pre-trained model `clf`
-    if model_path is not None:
-        model = load_scikit_model(model_path.values[0])
-    else:
-        curdir = os.path.dirname(os.path.abspath(__file__))
-        model_path = curdir + '/data/models/kn_diff_pc_sets.pkl'
-        model = load_scikit_model(model_path)
+    # Load pre-trained model
+    model = load_classifier("complete.pkl")
```

Comment:
@JulienPeloton , can we somehow take this file name as input? 
it is saved within kndetect and is directly loading the classifier saved in kndetect

Reply:
Yes -- see the previous syntax of the module:

```python
def knscore(jd, fid, magpsf, sigmapsf, jdstarthist, cdsxmatch, ndethist, model_path=None, pcs_path=None, npcs=None) -> pd.Series:
...
  # Load pre-trained model `clf`
    if model_path is not None:
        model = load_scikit_model(model_path.values[0])
    else:
        curdir = os.path.dirname(os.path.abspath(__file__))
        model_path = curdir + '/data/models/kn_diff_pc_sets.pkl'
        model = load_scikit_model(model_path)
   ...
```

same for pcs file.

Reply:
So you just need to have `model_path=None, pcs_path=None, npcs=None` in the constructor, and define the default behaviour in the function.

Reply:
I just made the changes and tried to upload it. But there seems to be a problem...
I am not sure where this is coming from (maybe due to the rebase?) because the error is in a line number that does not exist in my PR
Can you help me out with this?

Reply:
hum, that's maybe my fault -- I changed the default branch. I restored it as before. Can you trigger again the CI with a dummy commit?